api/envoy/extensions/filters/http/buffer/v3/buffer.proto

```diff
@@ -28,6 +28,11 @@ message Buffer {
   // manager will stop buffering and return a 413 response.
   google.protobuf.UInt32Value max_request_bytes = 1
       [(validate.rules).uint32 = {gt: 0}, (validate.rules).message = {required: true}];
+
+  // Set the maximum request size but doesn't buffer the data immediately.
```

Comment:
should "immediately" be replaced with "doesn't buffer the data in the Buffer filter"?